docs/Usage/Concert/Concert.md

```diff
@@ -0,0 +1,224 @@
+## Requirements
+### Part 1: Mental Imagery
+TODO
+
+### Part 2: Color/Emotion - App-Only
+| Emotions          | Artist       | Song                                                 | Duration |
+|-------------------|--------------|------------------------------------------------------|----------|
+| Serenity/Calm     | Fauré        | Pavane - initial excerpt                             | 2:30     |
+| Sadness/Nostalgia | Grieg        | Elegiac Melodies - No. 2, Last Spring excerpt        | 2:00     |
+| Happiness/Joy     | Beethoven    | Symphony No. 7 - 4th movement - finale excerpt.      | 2:00     |
+| Anger             | Shostakovich | Chamber Symphony in C minor, Op. 110a - 2nd movement | 3:00     |
+
+* The audience will listen to the orchestra play the above four pieces of music.
+* Each audience member will use the app to pick from six available colors (RGB, CMY).
+* It's possible that we will want audience members of the final rehearsal to be able to choose emotions instead.
+* The audience member can pick a new color at any time, but the app will give a gentle reminder every 20 seconds.
+* During the intermission (after the four songs), each audience member's app will display both their own choices as well as the overall audience distribution. Let's call this the "results" screen.
+* The app will move from one song to the next as the event progresses without user input
+
+### Part 3: Color/Emotion - App + Central Visualization
+* The audience will listen to the orchestra play the Coriolan overture.
+* The audience will be randomly split into two groups.
+    * In group A, the functionality is the same as the app-only section.
+    * In group B, the audience member will instead be picking emotions: Joy, sadness, anger, fear, [disgust].
+* There will also be a central display in the event venue visualizing the audience's choices as they come in. This should function like a "mood ring" in the sense that it shows the most recent audience choices.
+
+### Part 4: Chills
+TODO
+
+## Authentication
+Use AWS Cognito libraries.
+
+## Websocket API
+### Report a choice
+Send `CHOICE_MADE`
+```js
+{
+  choiceType: String,           // CHOICE_COLOR || CHOICE_EMOTION || CHOICE_CHILLS || CHOICE_IMAGERY
+  choice: String || Boolean,    // Boolean for chills, String for the rest
+  timestamp: String,
+}
```

Comment:
Thinking about displaying users' choices back to them during the intermission. Do we need some kind of userId (access token?) in here? Or will we store user choices in memory or get them some other way?

Reply:
The mobile app can store the users' choices, I don't see a need for the back-end to repeat them back.

Also, the first thing the app does when opening the websocket connection is send a JWT.  At that point, the server can save user ID as a property on the connection object. So either way, we're covered.

Reply:
Got it. Storing them in the app was what I meany by in memory. Works for me.